docs/helpers/Triggers.md

```diff
@@ -0,0 +1,109 @@
+| [IsEqualStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.IsEqualStateTrigger) | Enables a state if the value is equal to another value |
+| [IsNotEqualStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.IsNotEqualStateTrigger) | Enables a state if the value is not equal to another value |
+| [IsNullOrEmptyStateTriggers](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.IsNullOrEmptyStateTriggers) | Enables a state if an Object is null or a String/IEnumerable is empty |
+| [NetworkConnectionStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.NetworkConnectionStateTrigger) | Trigger for switching when the network availability changes |
+| [RegexStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.RegexStateTrigger) | Enables a state if the regex expression is true for a given string value |
+| [UserHandPreferenceStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.UserHandPreferenceStateTrigger) | Trigger for switching UI based on whether the user favors their left or right hand |
+| [UserInteractionModeStateTrigger](https://docs.microsoft.com/dotnet/api/Microsoft.Toolkit.Uwp.UI.Triggers.UserInteractionModeStateTrigger) | Trigger for switching when the User interaction mode changes (tablet mode) |
+
+## CompareStateTrigger Example
+```
+<VisualState.StateTriggers>
+    <triggers:CompareStateTrigger Value="{Binding Value,ElementName=Slider, Mode=OneWay}" To="3" Comparison="LessThan" />
+</VisualState.StateTriggers>
+```
+## FullScreenModeStateTrigger Example 
+```
+<VisualState.StateTriggers>
+    <triggers:FullScreenModeStateTrigger IsFullScreen="true" />
+</VisualState.StateTriggers>
+```                    
+## IsEqualStateTrigger Example
+```
+<VisualState.StateTriggers>
+    <triggers:IsEqualStateTrigger Value="{Binding IsChecked, ElementName=checkbox, Mode=OneWay}" To="{x:Null}" />
+</VisualState.StateTriggers>
+```
+## IsNotEqualStateTrigger Example
+```
+<VisualState.StateTriggers>
+    <triggers:IsNotEqualStateTrigger Value="{Binding IsChecked, ElementName=checkbox, Mode=OneWay}" To="{x:Null}" />
+</VisualState.StateTriggers>
+```
+## IsNullOrEmptyStateTriggers Example
+```
+<VisualState.StateTriggers>
+    <triggers:IsNullOrEmptyStateTrigger Value="{Binding Text, ElementName=OurTextBox, Mode=OneWay}"/>
+</VisualState.StateTriggers>
+```              
+## NetworkConnectionStateTrigger Example
+```
+<VisualState.StateTriggers>
+    <triggers:NetworkConnectionStateTrigger ConnectionState="Connected" />
+</VisualState.StateTriggers>
+```
+## RegexStateTrigger Example
+```
+<VisualState.StateTriggers>
+    <triggers:RegexStateTrigger Value="{Binding Text, ElementName=emailTextBox, Mode=OneWay}"
+		                    Expression="^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$"
+				    Options="IgnoreCase" />
```

Comment:
Think there's a mix of tabs and spaces here that was causing issues with alignment in the page:

```suggestion
    <triggers:RegexStateTrigger Value="{Binding Text, ElementName=emailTextBox, Mode=OneWay}"
                                                  Expression="^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$"
                                                  Options="IgnoreCase" />
```